webapp/locales/en.json

```diff
@@ -145,7 +145,8 @@
     },
     "comment": {
       "submit": "Comment",
-      "submitted": "Comment Submitted"
+      "submitted": "Comment Submitted",
+      "updated": "Comment Updated"
```

Comment:
@ALau2088 the German translation for `Commented Updated` would be `Kommentar Aktualisiert`. Although I have to admit this sounds a little strange to me. Probably I would use a more generic wording `Changes Saved` <-> `Änderungen gespeichert`.